src/components/Modal/index.styles.js

```diff
@@ -109,16 +171,26 @@ const bottomActionBarGutters = css`
 
   ${largeAndUp`
     padding: ${spacing.gutters.largeAndUp}px;
-    padding-top: 24px;
+    padding-top: ${spacing.gutters.mediumAndUp}px;
   `};
 `;
 
 export const BottomActionBar = styled.div`
   position: relative;
+  overflow: hidden;
   box-shadow: ${({ shadow }) =>
     shadow ? `${SHADOW_OFFSET_X}, ${SHADOW_OFFSET_Y}` : "none"};
 
   ${({ gutters }) => (gutters ? bottomActionBarGutters : "")};
+
+  button {
```

Comment:
Can you please either make this selector more specific or implement it in the catalog example?

Reply:
This is basic styling for child buttons of the component ( implemented if no specific styles are provided) not for buttons in general.

Reply:
Ok